mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -591,6 +591,7 @@ ForOp::replaceWithAdditionalYields(RewriterBase &rewriter,
   // Replace the old loop.
   rewriter.replaceOp(getOperation(),
                      newLoop->getResults().take_front(getNumResults()));
+  newLoop->setAttrs(getOperation()->getAttrs());
```

Comment:
Is this relevant for this change? Looks like an omission in the original implementation of this operation, but we need to be careful and avoid carrying over inherent attributes, so it would need a separate test.